File: sqlbuilder/sqlbuilder/Compare.java
package com.clickability.cms.dataaccess.sqlbuilder;

public class Compare extends AbstractCompare<Compare> {
	
	public Compare() {
		super();
	}
	
	public Compare(String lhs, String op) {
		this();
		lhs(lhs);
		using(op);
	}
	
	public Compare(String lhs, String op, String rhs) {
		this(lhs, op);
		rhs(rhs);
	}

}

File: java/queryBuilder/Compare.kt
package com.clickability.cms.dataaccess.sqlbuilder

class Compare() : AbstractCompare<Compare>() {

    constructor(lhs: String, op: String) : this() {
        lhs(lhs)
        using(op)
    }

    constructor(lhs: String, op: String, rhs: String) : this(lhs, op) {
        rhs(rhs)
    }

}
